sdk/textanalytics/Azure.AI.TextAnalytics/src/MinedOpinion.cs

```diff
@@ -12,19 +12,19 @@ namespace Azure.AI.TextAnalytics
     /// </summary>
     public readonly struct MinedOpinion
     {
-        internal MinedOpinion(AspectSentiment aspect, IReadOnlyList<OpinionSentiment> opinions)
+        internal MinedOpinion(OpinionTargetSentiment aspect, IReadOnlyList<OpinionDescriptionSentiment> opinions)
         {
-            Aspect = aspect;
-            Opinions = new List<OpinionSentiment>(opinions);
+            Target = aspect;
+            Descriptions = new List<OpinionDescriptionSentiment>(opinions);
```

Comment:
The idea here is that a MinedOpinion consists of an OptionTarget, and a set of OpinionDescriptions.  The target is the target of the description, and together, those make up an opinion.

Simple conceptual model:

Opinion:
- Target
- Descriptions

e.g.

Opinion: "the hotel was smelly and gross"
Target: "hotel"
Descriptions: { "smelly", "gross" }

Reply:
Conceptually, yes.